playgrounds/vanilla-js/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#000000" />
+
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/instantsearch.css@7/themes/algolia-min.css"
+    />
+    <link rel="stylesheet" href="./src/index.css" />
+    <link rel="stylesheet" href="./src/app.css" />
+
+    <title>MeiliSearch + InstantSearch</title>
+  </head>
+
+  <body>
+    <div class="ais-InstantSearch">
+      <h1>MeiliSearch + InstantSearch.js</h1>
+      <h2>Search in Steam video games 🎮</h2>
+      <p>
+        This is not the official Steam dataset but only for demo purpose. Enjoy
+        searching with MeiliSearch!
+      </p>
+
+      <div class="left-panel">
+        <div id="clear-refinements"></div>
+
+        <h2>Genres</h2>
+        <div id="genres-list"></div>
+        <h2>Players</h2>
+        <div id="players-list"></div>
+        <h2>Platforms</h2>
+        <div id="platforms-list"></div>
+        <h2>Misc</h2>
+        <div id="misc-list"></div>
+      </div>
+
+      <div class="right-panel">
+        <div id="searchbox" class="ais-SearchBox"></div>
+        <div id="hits"></div>
+        <div id="pagination"></div>
+      </div>
+    </div>
+
+    <script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch@v0.2.3/dist/instant-meilisearch.umd.min.js"></script>
```

Comment:
Can we use the local file instead and not the CDN one? Otherwise, there is no point in doing the playground. There is already the code-sambox to allow the user to play with the code. The playgrounds are for contributors.